### DISCUSSION TITLE
Convert EPS with clipping path to PNG reverses crop target and background color

### DISCUSSION THREAD

**Discussion by coffeesmell** [ImageMagick/ImageMagick#3934] [Help]
Is there any changes regarding clipping path in latest Imagemagick version 7.1.0-2?
When I convert an EPS file with clipping path to transparent PNG, the crop target was reversed in this latest version.

In older versions like Imagemagick version 7.0.8-6, conversion has no problem, the crop target was correct based on the clipping path.

This is the command I used for both versions.

`convert c:\test\Untitled-2.eps[0] -alpha transparent -clip -alpha opaque +clip -background transparent PNG32:c:\test\untitled-2_PNG.png`

This is the source EPS file:  https://drive.google.com/drive/folders/1ihHNxA8Vtq3uARJtJFUthm6hzsl0rWOV?usp=sharing

Below is conversion result.

Imagemagick version 7.0.8-6
![image](https://user-images.githubusercontent.com/87455212/125719351-caa0aaa8-d5ce-400c-8c25-41990295e458.png)

 Imagemagick version 7.1.0-2
![image](https://user-images.githubusercontent.com/87455212/125719234-e4c33572-e1c6-4533-8cd3-1cb8fbf2671f.png)

**Comment by coffeesmell** [comment]
Is this a bug of latest ImageMagick version？　
Can someone confirm this？

**Comment by dlemstra** [comment]
Added file to question: [Untitled-2.zip](https://github.com/ImageMagick/ImageMagick/files/6987670/Untitled-2.zip)

**Comment by dlemstra** [comment]
Your file contains the following clip mask:

![clip](https://user-images.githubusercontent.com/10426229/129470643-d513e1a1-0fe5-4b9b-8218-faa9e2469d27.png)

The black part of the image will be changed to transparent. So yes the behaviour is correct. We probably fixed a bug in an earlier version.
